setup.cfg

```diff
@@ -79,6 +79,7 @@ fmriprep =
     data/boilerplate.bib
     data/itkIdentityTransform.txt
     data/flirtsch/bbr.sch
+    data/reports-spec.yml
```

Comment:
For this to correctly work, we might need to remove `include_package_data = True`. I've recently discovered (https://github.com/niflows/nipype1-workflows/pull/3, https://github.com/nipy/nibabel/commit/7120347111d0f0a2af29bb9ba2d133f61fdb17b8) that that line seems to mean *only* use `MANIFEST.in`.

Reply:
If I understand correctly, we either set `include_package_data = False` or add all these objects in the `MANIFEST.in`?

Reply:
I think so.

OTOH I think these must have worked previously...

Reply:
Moving the conversation to #1896